pkg/mem/mem.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * Minio Cloud Storage, (C) 2016, 2017, 2018 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mem
+
+import (
+	"runtime"
+)
+
+// Performance holds memory utilization information in human readable format
+type Performance struct {
+	Mem   string `json:"mem"`
+	Error string `json:"error,omitempty"`
+}
+
+// GetPerformance measures the total memory provisioned for the current process
+// from the OS
+func GetPerformance() Performance {
+	memStats := new(runtime.MemStats)
+	runtime.ReadMemStats(memStats)
```

Comment:
Is this performance?  we should simply call this MemoryStats() since performance would mean that we are testing `allocations` in some manner.

Reply:
These are stats. Will update the func signature